test/fixtures/topLevelAwait2.js

```
@@ -0,0 +1 @@
+export default await Promise.resolve(1);
```

We should throw error on this, it is invalid syntax, try to run it without webpack

---

Ok, I've changed the example to declare the value separately above 👍.